packages/hoppscotch-common/src/newstore/settings.ts

```diff
@@ -119,10 +119,8 @@ export const getDefaultSettings = (): SettingsDef => {
 
   // Wait for platform to initialize before setting CURRENT_INTERCEPTOR_ID
   nextTick(() => {
-    applySetting(
-      "CURRENT_INTERCEPTOR_ID",
+    defaultSettings.CURRENT_INTERCEPTOR_ID =
       platform?.interceptors.default || "browser"
-    )
   })
```

Comment:
Is it necessary to set the default value here since the interceptor module accounts for it already?
cc @AndrewBastin 

https://github.com/hoppscotch/hoppscotch/blob/f8ac6dfeb127d37696d29adc81455dec65572702/packages/hoppscotch-common/src/modules/interceptors.ts#L26-L44